test/performance/range/views/view_translate_1D_benchmark.cpp

```diff
@@ -37,23 +36,25 @@ template <typename rng_t>
 void sequential_read_impl(benchmark::State & state, rng_t && rng)
 {
     for (auto _ : state)
-        for (aa27 c : rng)
+        for (seqan3::aa27 c : rng)
             benchmark::DoNotOptimize(c.to_rank());
 }
 
 template <typename tag_t>
 void sequential_read(benchmark::State & state)
 {
-    std::vector<dna4> dna_sequence{test::generate_sequence<seqan3::dna4>(1000, 0, 0)};
+    std::vector<seqan3::dna4> dna_sequence{seqan3::test::generate_sequence<seqan3::dna4>(1000, 0, 0)};
 
     if constexpr (std::is_same_v<tag_t, baseline_tag>)
     {
-        std::vector<aa27> translated_aa_sequence = dna_sequence | views::translate_single | views::to<std::vector<aa27>>;
+        std::vector<seqan3::aa27> translated_aa_sequence = dna_sequence
+                                                         | seqan3::views::translate_single
+                                                         | seqan3::views::to<std::vector<seqan3::aa27>>;
```

Comment:
I had to look this up, I didn't know that we allow this style. I would personally prefer this stye:

```cpp
std::vector<seqan3::aa27> translated_aa_sequence = dna_sequence | seqan3::views::translate_single
                                                                | seqan3::views::to<std::vector<seqan3::aa27>>;
```